src/services/user/hooks/global.js

```diff
@@ -0,0 +1,19 @@
+const mailToLowerCase = (hook) => {
+	if (hook.data) {
+		if (hook.data.email) {
+			hook.data.email = hook.data.email.toLowerCase();
```

Comment:
there should be one single cleanup method that is reused on all selected properties if they exist. trim() should also be a nice string function

Reply:
what do you mean with cleanup method? Sounds like a other solition, should be added